packages/stellar_client/lib/src/client.dart

```diff
@@ -0,0 +1,440 @@
+      return false;
+    }
+  }
+
+  Future<bool> addTrustLine() async {
+    for (var entry in _currencies.currencies.entries) {
+      String currencyCode = entry.key;
+      currency.Currency Currency = entry.value;
+
+      String issuerAccountId = Currency.issuer;
+      Asset currencyAsset =
+          AssetTypeCreditAlphaNum4(Currency.assetCode, issuerAccountId);
+
+      // TODO:
+      ChangeTrustOperationBuilder changeTrustOperation =
+          ChangeTrustOperationBuilder(currencyAsset, "300000");
+
+      final account = await _sdk.accounts.account(accountId);
+
+      Transaction transaction = TransactionBuilder(account)
+          .addOperation(changeTrustOperation.build())
+          .build();
+      transaction.sign(_keyPair, _stellarNetwork);
+
+      SubmitTransactionResponse response =
+          await _sdk.submitTransaction(transaction);
+
+      if (!response.success) {
+        print("Failed to add trustline for $currencyCode");
+        return false;
+      } else {
+        print("trustline for $currencyCode was added successfully");
+        return true;
+      }
+    }
+
+    print("No trustlines were processed");
+    return false;
+  }
+
+  Future<bool> sendTransaction(
+      {required String destinationAddress,
+      required String amount,
+      required String currency,
+      String? memoText}) async {
+    try {
+      final receiver = await _sdk.accounts.account(accountId);
+
+      var specificBalance = receiver.balances.firstWhere(
+        (balance) => balance.assetCode == currency,
+        orElse: () {
+          throw Exception('Balance with asset code $currency not found.');
+        },
+      );
+
+      final asset = _currencies.currencies[currency];
+      if (asset == null) {
+        throw Exception('$currency not supported');
+      }
+
+      AccountResponse sender = await _sdk.accounts.account(accountId);
+
+      Asset tftAsset = AssetTypeCreditAlphaNum4(asset.assetCode, asset.issuer);
+
+      Transaction transaction = TransactionBuilder(sender)
+          .addOperation(
+              PaymentOperationBuilder(destinationAddress, tftAsset, amount)
+                  .build())
+          .addMemo(memoText != null ? Memo.text(memoText) : Memo.none())
+          .build();
+
+      transaction.sign(_keyPair, _stellarNetwork);
+      await _sdk.submitTransaction(transaction);
+      print("Transaction successful.");
+      return true;
+    } catch (error) {
+      print("Failed to send transaction: $error");
+      return false;
+    }
+  }
+
+  Future<bool> activateThroughWallet({required String secretSeed}) async {
+    try {
+      KeyPair walletKeyPair = KeyPair.fromSecretSeed(secretSeed);
+      AccountResponse existingAccount =
+          await _sdk.accounts.account(walletKeyPair.accountId);
+
+      Transaction transaction = TransactionBuilder(existingAccount)
+          .addOperation(CreateAccountOperationBuilder(accountId, "10").build())
```

Comment:
10 is a bit much. AFAIR we were used to transfer 2.6 XLM to activate the wallet. Can you try that or less? we should use the minimal value